emcmake.py

```diff
@@ -30,7 +31,7 @@ def has_substr(args, substr):
     return any(substr in s for s in args)
 
   # Append the Emscripten toolchain file if the user didn't specify one.
-  if not has_substr(args, '-DCMAKE_TOOLCHAIN_FILE'):
+  if not has_substr(args, '-DCMAKE_TOOLCHAIN_FILE') and not os.environ.get("CMAKE_TOOLCHAIN_FILE"):
```

Comment:
I think this can be just: `'CMAKE_TOOLCHAIN_FILE' not in os.environ` (note the single quotes)

Reply:
Updated, thanks!